### PR TITLE
User Dashboard

### PR DESCRIPTION
**Pull request by DanWilkins2107**

### CODE REVIEW

File: screens/LoginScreens/ForgottenPasswordScreen.jsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ export default function ForgottenPasswordScreen({ navigation }) {
             </Text>
             <Button title = "Back to Login"
                 onPress = {() => navigation.navigate("Login")} />
+            <Button title = "TEMPORARY: Go to User Dashboard"
+                onPress = {() => navigation.navigate("UserDashboard")} />
```
Comment (dwulfse) [inline screens/LoginScreens/ForgottenPasswordScreen.jsx:12]: This has got to go then integrate back in once login screens are in

File: screens/UserScreens/UserDashboard.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                </CreateAndJoinButtons>
+                <CreateAndJoinButtons title="Join">
+                    <FontAwesomeIcon icon={faMasksTheater} size={50} />
+                </CreateAndJoinButtons>
+            </View>
+            <View className="border-b-2">
+                <Text className="text-3xl font-extrabold text-center mb-5">Your Productions</Text>
+            </View>
+            <ScrollView className="flex-col space-y-12 grow">
+                <ProductionButton {...mockProduction} />
```
Comment (dwulfse) [inline screens/UserScreens/UserDashboard.jsx]: Maybe make an array elsewhere of mock productions and add the actual code to iterate over them and display them all

File: screens/UserScreens/UserDashboard.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                    <FontAwesomeIcon icon={faClapperboard} size={50} />
+                </CreateAndJoinButtons>
+                <CreateAndJoinButtons title="Join">
+                    <FontAwesomeIcon icon={faMasksTheater} size={50} />
+                </CreateAndJoinButtons>
+            </View>
+            <View className="border-b-2">
+                <Text className="text-3xl font-extrabold text-center mb-5">Your Productions</Text>
+            </View>
+            <ScrollView className="flex-col space-y-12 grow">
```
Comment (dwulfse) [inline screens/UserScreens/UserDashboard.jsx:22]: ScrollView offers a StickyHeaderComponent method  which could be better than the text header above

Comment (DanWilkins2107) [reply screens/UserScreens/UserDashboard.jsx:22]: Don't think it'll be any cleaner ngl

Comment (dwulfse) [reply screens/UserScreens/UserDashboard.jsx:22]: 100%

**Pull request closed without merging**